tests/test_instanceconnectionmanager_unit.py

```diff
@@ -46,3 +47,98 @@ def test_InstanceConnectionManager_wrong_connection_string():
     loop = asyncio.new_event_loop()
     with pytest.raises(CloudSQLConnectionError):
         InstanceConnectionManager("test-project:test-region", loop)
+
+
+def test_InstanceConnectionManager_get_ephemeral():
+    """
+    Test to check whether _get_ephemeral runs without problems given a valid
+    connection string.
+    """
+
+    try:
+        connect_string = os.environ["INSTANCE_CONNECTION_NAME"]
+    except KeyError:
+        raise KeyError(
+            "Please set environment variable 'INSTANCE_CONNECTION"
+            + "_NAME' to a valid Cloud SQL connection string."
+        )
+
+    loop = asyncio.new_event_loop()
+    icm = InstanceConnectionManager(connect_string, loop)
+
+    async def create_client_session():
+        return aiohttp.ClientSession()
+
+    fut_client_session = asyncio.ensure_future(create_client_session(), loop=loop)
+
+    icm._loop.run_until_complete(fut_client_session)
+
+    client_session = fut_client_session.result()
```

Comment:
L69-79 - Why cant we use `aiohttp.ClientSession(loop=loop)`?

Reply:
According to the `aiohttp` documentation, the `loop` keyword param looks like it's been deprecated